modules/perception/map/hdmap/hdmap_input.cc

```diff
@@ -94,11 +94,12 @@ bool HDMapInput::InitHDMap() {
     AERROR << "Failed to find hadmap file: " << hdmap_file_;
     return false;
   }
-  if (hdmap_->LoadMapFromFile(hdmap_file_) != 0) {
+  if (!hdmap_->LoadMapFromFile(hdmap_file_)) {
```

Comment:
Wrong logic?

Reply:
Right. will fix it and following same issue soon.